surveys/2024-annual-survey/translations/fr.md

```diff
@@ -65,37 +65,37 @@ Type: select one
 
 Type: select one
 
-- J'essaie toujours activement d'apprendre Rust.
-- Courant 2023
-- Courant 2021 ou 2022
-- Courant 2019 ou 2020
-- Courant 2017 ou 2018
-- Courant 2016 ou avant
+- J’essaie toujours activement d’apprendre Rust.
+- Courant 2024
+- Courant 2022 ou 2023
+- Courant 2020 ou 2021
+- Courant 2018 ou 2019
+- Courant 2017 ou avant
 
-### Si vous avez consommé du matériel d’apprentissage sur Rust, quel type de matériel avez-vous consommé ?
+### Si vous avez consommé du matériel d’apprentissage sur Rust, quel type de matériel avez-vous consommé ?
 
 Type: select all that apply
 
-- Livres (&quot;Le langage de programmation Rust&quot;, &quot;Rust for Rustaceans&quot;, etc.)
-- Exercices en ligne (Rustlings, Rust by Exemple, etc.)
-- Vidéos
-- 
-- 
-- 
+- Livres (« Le langage de programmation Rust », « Rust for Rustaceans », etc.)
+- Exercices en ligne (Rustlings, 100 Exercises To Learn Rust, etc.)
+- Vidéos ou diffusions en direct
+- Blogs
+- Docunentation
```

Comment:
Perhaps this should be Documentation?

Reply:
Fixed it in my branch, will create PR soon. Thanks!